packages/atomic/src/utils/initialization-utils.tsx

```diff
@@ -177,22 +188,23 @@ export function InitializeBindings<SpecificBindings extends AnyBindings>({
           }
         }
       );
+      if (element.isConnected) {
```

Comment:
Except this; Again, ZoneJS is sometimes very confused and will run stencil lifecycle method out of whack (for example, willLoad before connectedCallback)